ethereum/gateway-eth-ts/src/commands/add-network-authority.ts

```diff
@@ -35,6 +35,7 @@ export default class AddNetworkAuthority extends Command {
       required: true,
       description:
         "Network authority address to add to the GatewayToken contract",
+      // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Confused this says it returns as Promise<string> but also returns `null`.  What are you ignoring here?